source/watchdog.py

```diff
@@ -292,7 +292,9 @@ def initialize():
 	NVDAHelper._setDllFuncPointer(NVDAHelper.localLib, "_notifySendMessageCancelled", _notifySendMessageCancelled)
 	_watcherThread = threading.Thread(
 		name=__name__,
-		target=_watcher
+		target=_watcher,
+		# TODO: should we change this? does this need to be kept alive to handle crashes?

```

Comment:
I think this is fine. Watchdog should already be terminated at process exit, unless there is some kind of logic error, which in that case we want the thread to die anyway.

Reply:
```suggestion
```